src/resilience/context/Context.cpp

```diff
@@ -70,7 +74,48 @@ namespace KokkosResilience
   std::unique_ptr< ContextBase >
   make_context( const std::string &config )
```

Comment:
Since filenames are now in the config file (which I support), we need a new way to get unique filenames per process. We could just have the user pass a pid in, then the StdFileContext can append ".<pid>" to the filename being passed to the StdFileBackend
```suggestion
  make_context( const int process_id, const std::string &config )
```

Edit: I just caught that you can just edit the Config object to set a unique filename, so this isn't totally necessary.